developer/src/kmldmlc/src/keyman/kmx/kmx-plus.ts

```diff
@@ -10,6 +10,37 @@ export class Section {
 }
 
 // 'sect'
+
+export type Sect = Section;
+
+// 'bksp' -- see 'tran'
```

Comment:
Some of these transform-related definitions should have gone into #7212 but splitting them out is probably not a worthwhile exercise at this point.